test/ipc_os_prov_shm.sh

```diff
@@ -20,7 +20,7 @@ UMF_LOG_VAL="level:debug;flush:debug;output:stderr;pid:yes"
 rm -f /dev/shm/${SHM_NAME}
 
 echo "Starting ipc_os_prov_shm CONSUMER on port $PORT ..."
-UMF_LOG=$UMF_LOG_VAL ./umf_test-ipc_os_prov_consumer $PORT $SHM_NAME &
```

Comment:
should you also update `ipc_ipcapi` example?

Reply:
The IPC example was written better :) but I changed the bash file that runs it